### ISSUE TITLE
Throw panic on traefik v2.5.5

### ISSUE THREAD

**Issue by NikonNLG** [XciD/traefik-plugin-rewrite-headers#2]
On new traefik rewrite headers plugin is failed with
```
ERRO[2021-12-17T17:54:34Z] plugins-local/src/github.com/XciD/traefik-plugin-rewrite-headers/rewrite_headers.go:66:3: panic  module=github.com/XciD/traefik-plugin-rewrite-headers plugin=plugin-rewriteHeaders
ERRO[2021-12-17T17:54:34Z] Recovered from panic in HTTP handler [172.17.0.1:63834 - /test]: plugins-local/src/github.com/XciD/traefik-plugin-rewrite-headers/rewrite_headers.go:70:19: method not found: Header  middlewareType=Recovery middlewareName=traefik-internal-recovery
```
Same configuration works with traefik 2.5.4, so looks like they change some dependencies.

Can you please check what's going on? Unfortunately, I'm newbie in golang, and can't help with it

**Comment by XciD** [comment]
Related to that
https://github.com/traefik/yaegi/issues/1335

**Comment by XciD** [comment]
Fixed in `v0.0.3`
